challenges/identification/I3_benchmark/I3_specification.md

```diff
@@ -0,0 +1,113 @@
+# I3 benchmark specification
+
+## Synopsis
+
+Benchmark to compare output of tool's RNAseq based site identification to an established atlas of polyA-sites (PolyASite, PolyA_DB). The goal is to calculate precision with respect to data from existing databases for several different distance thresholds.
+
+Input data:
+
+1. Poly(A) sites identified based on RNAseq data using the benchmarked tool
+2. Poly(A) sites identified in PolyASite database
+3. Poly(A) sites identified in PolyA_DB database
+
+Based on the input data the following metrics are computed:
+
+1. Precision = (TP/(TP+FP))
+
+TP - true positives - PAS identified by the tool and present within X nucleotides of PAS identified in the database  
+FP - false positives  - PAS identified by the tool and not present within X nucleotides of PAS identified in the database
+
+The metrics should be calculated for the following thresholds of distance between PAS identified by the tool and PAS present in databases:
+
+- 50 nt
```

Comment:
Do we also want this expanded to a sequence from 0 to 200 nt using 10 nt steps? Or maybe 20 nt steps?

Reply:
For the other ones we've used 0-100nt with 10nt step - we can do the same here or is there a reason to use 0-200nt range (and should we change it to 200 for other benchmarks too)?

Reply:
I'm fine either way. We could change I2 to 200 nt just to be consistent. Although, I kind of expect 100 nt on the ground truth might be sufficiently informative since it's coming from the same biological sample, whereas here we are comparing against other experimental results.

Reply:
I'm thinking now, should we have distances in a separate input file so the code is a bit more flexible?